src/Controls/GlobeControls.js

```diff
@@ -348,11 +358,7 @@ class GlobeControls extends THREE.EventDispatcher {
         }
     }
 
-    dollyIn(dollyScale) {
-        if (dollyScale === undefined) {
-            dollyScale = this.dollyScale;
-        }
-
+    dolly(dollyScale = 1) {
```

Comment:
What is the case where is undefined?

Reply:
Doesn't happen in iTowns code. I changed the method signature which now take a `delta` in argument. The method will dolly in or out according to the sign of `delta`.